flaskbb/forum/views.py

```diff
@@ -1060,182 +1061,215 @@ def post(self, post_id):
         return redirect(post.topic.url)
 
 
+class MarkdownPreview(MethodView):
+    decorators = [login_required]
+
+    def post(self, mode=None):
+        text = request.data.decode("utf-8")
+
+        if mode == "nonpost":
+            render_cls = current_app.pluggy.hook.\
+                flaskbb_load_nonpost_markdown_class(
+                    app=current_app
+                )
+        else:
+            render_cls = current_app.pluggy.hook.\
```

Comment:
nitpick: This isn't the class itself, it's a collection of renderer mixin classes.

Reply:
I was looking for a shorter version of the var to not go over the line limit and have to break the function call into another line (I know this is dumb.. :P) but in the end it didn't matter anyway - changed it back to ``renderer_classes``.